HeterogeneousCore/AlpakaInterface/interface/traits.h

```diff
@@ -0,0 +1,69 @@
+#ifndef HeterogeneousCore_AlpakaInterface_interface_traits_h
+#define HeterogeneousCore_AlpakaInterface_interface_traits_h
+
+#include <type_traits>
+
+#if __cplusplus >= 202002L
+namespace cms {
+  using std::is_bounded_array;
+  using std::is_unbounded_array;
+}  // namespace cms
+#else
+#include <boost/type_traits/is_bounded_array.hpp>
+#include <boost/type_traits/is_unbounded_array.hpp>
```

Comment:
In principle these imply `<use name="boost_header"/>`(?) for `BuildFile.xml` until we have fully enabled C++20 (or so?).

Reply:
True (on the other hand, `alpaka` already depends on `boost`), I will add it to the BuildFile.